packages/core/data-transfer/src/file/providers/source/index.ts

```diff
@@ -184,7 +184,7 @@ class LocalFileSourceProvider implements ISourceProvider {
               return false;
             }
 
-            const parts = path.relative('.', filePath).split('/');
+            const parts = path.relative('.', filePath).split('\\');
```

Comment:
Doing this will break the feature on all OS except Windows. We might want to look into another solution here :eyes:

Reply:
as you said i also think path.sep will work for both kind of os.......can you elaborate more about archive issue by a scenario ?